Source/OCcore/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ endif()
 #endforeach()
 #list(REMOVE_DUPLICATES INCLUDE_DIRS)
 
-ADD_LIBRARY(${PROJECT_NAME} SHARED ${SOURCE_FILES} ${PLATFORM_FILES})
+ADD_LIBRARY(${PROJECT_NAME} SHARED ${SOURCE_FILES} ${PLATFORM_FILES} RawDump/rawdump.h RawDump/rawdump.cpp)

```

Comment:
Why are this files added manually, as everything should be retrieved automatically? They should be moved to some specific folder, maybe also /Image.

Reply:
I am sorry for this, CLion defaults added these.